e2e/playwright/various.spec.ts

```diff
@@ -449,45 +431,45 @@ test('Delete key does not navigate back', async ({ page }) => {
   const settingsCloseButton = page.getByTestId('settings-close-button')
 
   await settingsButton.click()
-  await expect(page.url()).toContain('/settings')
+  await expect.poll(() => page.url()).toContain('/settings')
```

Comment:
Oh nice I bet that wasn't auto-awaiting this is a great use of `poll`!

Reply:
Yep 100% that